app/Relations/LinkedPhotoCollection.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Relations;
+
+use App\Models\Configs;
+use App\Models\Photo;
+use Illuminate\Database\Eloquent\Collection;
+
+/**
+ * Class LinkedPhotoCollection.
+ *
+ * If serialized to JSON, each element of the resulting array contains two
+ * JSON attributes which link to the previous and next element.
```

Comment:
IMHO, it does belong to the front-end. My Android app (whenever it will see the light of world) won't rely on that, but simply take the list of photos (as if they were not linked) and then decide itself if there is a wrap-around or not.

However, writing front-ends with JS is out of my wheelhouse. So I tried to not change the front-end.

Reply:
Sounds like we are in agreement. I agree: not worth changing the behavior _right now_.